dpos/p2p/server.go

```diff
@@ -811,9 +823,16 @@ func NewServer(origCfg *Config) (*server, error) {
 		return nil, err
 	}
 
+	admgr := addrmgr.New(cfg.DataDir)
+	var hubService *hub.Hub
+	if cfg.EnableHub {
```

Comment:
If EnableHub is not configurable, there should be no conditional statement here.

Reply:
This config is designed to use in code level, for DPOS package is not only used by Elastos.ELA and also Elastos.Aribter or other projects. Elastos.ELA DPOS network provides hub service for other services is a feature and must enabled, but Elastos.Aribter or other projects can decide enable or disable hub service by their needs.